KsApi/lib/Route.swift

```diff
@@ -243,6 +248,9 @@ internal enum Route {
       case .userSelf:
         return (.GET, "/v1/users/self", [:], nil)
 
+      case let .userSelfWithToken(token):
+        return (.GET, "/v1/users/self", ["oauth_token": token], nil)
```

Comment:
Parameters passed in here will override the defaults, which in this case will be missing the `oauth_token`.